NEWS

```diff
@@ -81,7 +81,7 @@ PHP                                                                        NEWS
     iterable. (timwolla)
 
 - Standard:
-  . Fixed empty array returned by str_split on empty input. (Michael Vorisek)
+  . Fixed str_split to return an empty array instead of a non-empty array on empty input. (Michael Vorisek)
```

Comment:
This needs to refer to #8924. That ticket is labeled as bug, but I rather think it is a feature request, so maybe:
```suggestion
    . Implemented FR GH-8924 (str_split should return empty array for empty
      string). (Michael Vorisek)
```
Note that lines in NEWS should not exceed 79 or 80 characters.